reprohack_hub/static/vendor/node_modules/@fortawesome/fontawesome-free/package.json

```diff
@@ -1,58 +1,87 @@
 {
-  "description": "The iconic font, CSS, and SVG framework",
-  "keywords": [
-    "font",
-    "awesome",
-    "fontawesome",
-    "icon",
-    "svg",
-    "bootstrap"
+  "_args": [
+    [
+      "@fortawesome/fontawesome-free@5.15.4",
+      "/Users/Anna/Documents/workflows/Reprohack-org/reprohack_site"
```

Comment:
@twinkarma is this right?

Reply:
It probably shouldn't be doing that, but at the same time it shouldn't really matter as we're only using the static assets from the library